### PR TITLE
Added Dangerfile for ruby

### PR DESCRIPTION
**Pull request by saantiaguilera**
As title describes, Im adding Dangerfile (http://danger.systems/) to have the file language detected as Ruby.

Some examples:

- https://github.com/danger/danger/blob/master/Dangerfile
- https://github.com/artsy/eigen/blob/master/Dangerfile
- https://github.com/cocoapods/cocoapods/blob/master/Dangerfile
- https://github.com/realm/jazzy/blob/master/Dangerfile

And a lot more :P

### CODE REVIEW

**Comment by saantiaguilera** [comment]
Its failing because of a whitelisted project :(

```
  1) Failure:
TestGrammars#test_whitelisted_submodules_dont_have_licenses [/home/travis/build/github/linguist/test/test_grammars.rb:136]:
The following whitelisted submodules have a license:
* vendor/grammars/ant.tmbundle
Please remove them from the project whitelist..
--- expected
+++ actual
@@ -1 +1 @@
-{}
+{"vendor/grammars/ant.tmbundle"=>"241e5ddbb4423d792216783e9f668bd670b026e4"}
```

**Comment by Alhadis** [comment]
Hola @saantiaguilera,

This isn't your fault; it's an issue with one of Linguist's dependencies. See #3337; please wait until it's merged before trying again. =)

Gracias.

**Comment by brandonblack** [comment]
@saantiaguilera if you can update your pull request with the latest from master it should resolve the Ant related issue in the failing build.

**Comment by saantiaguilera** [comment]
cc @brandonblack @Alhadis 

Done :)

File: samples/Ruby/filenames/Dangerfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  fail "The core plugins are not at 100% doc'd - see below:", sticky: false
+  markdown "```\n#{core_lint_output}```"
+elsif core_lint_output.include? "warning"
+  warn "The core plugins are have yard warnings - see below", sticky: false
+  markdown "```\n#{core_lint_output}```"
+end
+
+junit.parse "junit-results.xml"
+junit.headers = [:file, :name]
+junit.report
```
Comment (brandonblack) [inline samples/Ruby/filenames/Dangerfile]: Missing a new line

**Comment by saantiaguilera** [comment]
@brandonblack Done :)

**Comment by brandonblack** [comment]
Thanks! 👍🏻

**Pull request merged**